### ISSUE TITLE
TypeError: this.cacheable is not a function

### ISSUE THREAD

**Issue by More4ever** [kisenka/svg-sprite-loader#184]
Hello

I've received an error on startup

```
TypeError: this.cacheable is not a function
    at new loader (node_modules/svg-sprite-loader/lib/loader.js:14:8)
```

Node: 8.3
npm: 5.3
Webpack: 3.4
OS: MacOS Sierra
Use with: Laravel Mix

Config:
```
mix.webpackConfig({
  module: {
    rules: [
      {
        test: /\.svg$/,
        loader: 'svg-sprite-loader',
        include: './resources/assets/img',
        options: {
          extract: true,
          spriteFilename: 'icons-sprite.svg'
        }
      }
    ],
  },
  plugins: [
    new webpack.ProvidePlugin({
      jQuery: 'jquery',
      jquery: 'jquery',
      $: 'jquery',
      'Popper': ['popper.js', 'default']
    }),
    new SpritePlugin(),
  ]
});
``` 

Stacktrace
```
TypeError: this.cacheable is not a function
    at new loader (node_modules/svg-sprite-loader/lib/loader.js:14:8)
    at Object.<anonymous> (webpack.mix.js:44:5)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/apple/projects/pdfio.dev/node_modules/laravel-mix/setup/webpack.config.js:9:1)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
```

**Comment by girwin575** [comment]
Is there any update to this? I'm having the same issue.

**Comment by kisenka** [comment]
@girwin575 sorry for the delay, just published `svg-sprite-loader@3.3.1` with fix.

**Comment by girwin575** [comment]
Thanks @kisenka will take a look!
